test/e2e-common.sh

```diff
@@ -87,9 +87,13 @@ function knative_teardown() {
 
 function knative_eventing() {
   # we need cert-manager installed to be able to create the issuers
-  kubectl apply -Rf "${CERTMANAGER_CONFIG}"
+  kubectl apply -f "${CERTMANAGER_CONFIG}/00-namespace.yaml"
 
-  wait_until_pods_running cert-manager || fail_test "Failed to setup cert-manager pods"
+  timeout 600 bash -c "until kubectl apply -f ${CERTMANAGER_CONFIG}/01-cert-manager.yaml; do sleep 5; done"
+  wait_until_pods_running "$CERT_MANAGER_NAMESPACE" || fail_test "Failed to install cert manager"
```

Comment:
seems `CERT_MANAGER_NAMESPACE` is not defined

```
Waiting until all pods in namespace  are uperror: flag needs an argument: 'n' in -n
See 'kubectl get --help' for usage.
.error: flag needs an argument: 'n' in -n
```

Reply:
Thanks, let's see https://github.com/knative-extensions/eventing-kafka-broker/pull/3598/commits/dffcb706dd797da53810f6c07c5f853db2c09583